### DISCUSSION TITLE
fail on compile frontend

### DISCUSSION THREAD

**Discussion by desenvolvedorjairomonassa** [apache/superset#19200] [Q&A / Help]
### Expected results

run frontend for installation of custom vizualisation

### Actual results

fail on compile

#### Screenshots

occurs when a tried to run forntend for instalationg of custom
the errors starting with ERROR in ./src/views/menu.tsx 29:95, complete error attached

#### How to reproduce the bug

1 - cd superset-frontend
2- yarn install
3 - npm run dev-servernode

### Environment

windows 10, using vscoding

- python version: python 3.9.6
- node.js version: v14.16.1

### Additional context

[error_complete.txt](https://github.com/apache/superset/files/6809465/error_complete.txt)

**Comment by srinify** [comment]
Two points!

- This may not be relevant, but as a heads up ... Superset isn't officially supported on Windows. One quick thing to try is to try this in a *nix environment and see if you run into the same issue.
- Have you made customizations to the frontend? It would help to know what customizations you made!
- What version of Superset is this? Are you running something close to `master` or a versioned release?

**Comment by RickYangzz** [comment]
npm ci

**Comment by nikhil-kuyya-talentas** [comment]
@Distringer as pointed need to use npm  not yarn in you second step.

**Comment by srinify** [comment]
Converting to Q&A since this is more a debugging thread than a bug or issue with Superset!
